taar/recommenders/recommendation_manager.py

```diff
@@ -27,6 +28,7 @@ def __init__(self, profile_fetcher=None, recommenders=None):
             self.recommenders = (
                 LegacyRecommender(),
                 CollaborativeRecommender(),
+                SimilarityRecommender(),
```

Comment:
This will fail as some changes are needed.

- The correct path needs to be provided [here](https://github.com/mozilla/taar/blob/0802bfad8aae984101b4b088850cebb25866e9ad/taar/recommenders/similarity_recommender.py#L7-L8). Change `DONOR_LIST_KEY` to point to `'taar/similarity/donors.json'` and `LR_CURVES_SIMILARITY_TO_PROBABILITY` to `'taar/similarity/lr_curves.json'`;
- Change [this](https://github.com/mozilla/taar/blob/0802bfad8aae984101b4b088850cebb25866e9ad/taar/recommenders/similarity_recommender.py#L43) to `donors['active_addons']`
- Change the [test](https://github.com/mozilla/taar/blob/master/tests/test_similarityrecommender.py#L13) accordingly: `activeAddons` -> `active_addons` and reference the constants from the similarity module [here](https://github.com/mozilla/taar/blob/master/tests/test_similarityrecommender.py#L77-L80) instead of typing them again.

Reply:
"ShaIl I rename all references inside the this loop to "donors" rather that "donor"? We are iterating through each donor individually, so I am wondering why the change?